core/src/main/java/org/apache/iceberg/rest/requests/UpdateTableRequest.java

```diff
@@ -31,7 +32,10 @@
 import org.apache.iceberg.relocated.com.google.common.collect.Lists;
 import org.apache.iceberg.relocated.com.google.common.collect.Sets;
 import org.apache.iceberg.rest.RESTRequest;
+import org.immutables.builder.Builder.Constructor;
+import org.immutables.value.Value;
 
+@Value.Style(newBuilder = "newBuilder")
```

Comment:
without this, one would instantiate the Builder via `new UpdateTableRequestBuilder()`, which is probably fine as well. We could also add
```
public static UpdateTableRequestBuilder builder() {
    return new UpdateTableRequestBuilder();
}
```
to `UpdateTableRequest`